build.sbt

```diff
@@ -167,9 +169,13 @@ import sbtassembly.AssemblyPlugin.defaultUniversalScript
 
 val scalacJvmOptions = Def.setting {
   val cross = CrossVersion.partialVersion(scalaVersion.value) match {
-    case Some((2, 13)) =>
-      Seq("-Ymacro-annotations", "-Xfatal-warnings", "-deprecation:false")
-    case _ => Seq.empty
+    case Some((2, 13)) => Seq(
+        "-Ymacro-annotations",
+        "-Xfatal-warnings",
+        "-deprecation:false",
+        "-target:8",
```

Comment:
1. same question: what's the difference between `-release 8` and `-target:8`?
2. also, are you setting it for all projects (including core), and not just dynamic and its two deps?

Reply:
Yes, I thought that's safest and it doesn't cause any issues really

Reply:
And looks it actually detected usage of a method added JDK 8 +